eng/scripts/stage-api-review-file.ps1

```diff
@@ -0,0 +1,50 @@
+param (
+  [Parameter(mandatory = $true)]
+  $PackageInfoPath,
+  [Parameter(mandatory = $true)]
+  $StagingDirectory
+)
+
+if (!((Test-Path $PackageInfoPath) -and (Test-Path $StagingDirectory)))
+{
+  Write-Error "Invalid parameter values. Pleaes verify values for these parameters."
+  exit 1
+}
+
+foreach($pkg in (Get-ChildItem -Path $PackageInfoPath "*.json"))
+{
+  $info = Get-Content -Path $pkg.FullName | ConvertFrom-Json
+  $apiFilePath = Join-Path $info.DirectoryPath "temp"
+  if (Test-Path $apiFilePath)
+  {
+    $apiFile = Get-ChildItem -Path $apiFilePath "*.api.json"
+    if ($apiFile)
+    {
+      if ($apiFile.Count -ne 1)
```

Comment:
Does this always give back a list? I know PS sometimes gives back a single object if there is only one in the list.

Reply:
I believe so. It was working when referring it directly without index also. I thought of adding this check to avoid any issue if we get any such scenario.